src/index.ts

```diff
@@ -70,25 +82,37 @@ export class PromiseQueue {
 			}
 
 			let timeout: ReturnType<typeof setTimeout> | undefined;
+			let enqueueTime: ReturnType<typeof process.hrtime>;
+			let serviceStartTime: ReturnType<typeof process.hrtime>;
```

Comment:
Can you declare this inside `wrappedFn` please, there's no need to scope it outside

Reply:
ack